base/src/test/java/uk/ac/ebi/atlas/model/ArrayDesignTest.java

```diff
@@ -0,0 +1,14 @@
+package uk.ac.ebi.atlas.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphanumeric;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ArrayDesignTest {
+    @Test
+    void unknownArrayDesignsArePopulatedWithTheirAccession() {
+        assertThat(ArrayDesign.createForUnknownName(randomAlphanumeric(20)))
+                .satisfies(arrayDesign -> assertThat(arrayDesign.accession()).isEqualTo(arrayDesign.name()));
```

Comment:
While I appreciate that AssertJ has the capability for this `satisfies` thing, I'm not sure it helps readability (or brevity)...